LayoutTests/fast/preloader/image-in-nested-template-01-expected.txt

```diff
@@ -0,0 +1 @@
+This test requires DumpRenderTree to see the log of what resources are loaded.
```

Comment:
It tends to be very helpful for test results to include pass/fail criteria. E.g. "this test passes if this is the only line in results, and fails if these are any loads logged.

Reply:
Let me explore if I can modify all these preload tests locally and if yes, I would add for all. :-)